Sources/WalletConnectSign/Services/CelanupService.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import WalletConnectKMS
+
+final class CleanupService {
+    
+    private let pairingStore: WCPairingStorage
+    private let sessionStore: WCSessionStorage
+    private let kms: KeyManagementServiceProtocol
+    
+    init(pairingStore: WCPairingStorage, sessionStore: WCSessionStorage, kms: KeyManagementServiceProtocol) {
+        self.pairingStore = pairingStore
+        self.sessionStore = sessionStore
+        self.kms = kms
+    }
+    
+    func cleanup() throws {
+        pairingStore.deleteAll()
+        sessionStore.deleteAll()
+        try kms.deleteAll()
+    }
+}
```

Comment:
There are other `KeyValueStore` in the session and pairing engine used for proposal-approval (even though they are more like workarounds). I think it would be good to clean those too.